Description: SKIA should be unconditional about compiling in ARM Neon
 optimizations, but only using them after runtime detection of CPU
 capabilities.
Origin: http://code.google.com/p/chromium/issues/detail?id=154107
Author: Chad Miller <chad.miller@canonical.com>

--- a/skia/skia_library_opts.gyp
+++ b/skia/skia_library_opts.gyp
@@ -98,7 +98,7 @@
             '../third_party/skia/src/opts/SkBitmapProcState_opts_arm.cpp',
           ],
         }],
-        [ 'target_arch == "arm" and (arm_version < 7 or (arm_neon == 0 and arm_neon_optional == 1))', {
+        [ 'target_arch == "arm" and (arm_version < 7 or arm_neon == 0)', {
           'sources': [
             '../third_party/skia/src/opts/memset.arm.S',
           ],
--- a/third_party/webrtc/system_wrappers/source/cpu_features.cc
+++ b/third_party/webrtc/system_wrappers/source/cpu_features.cc
@@ -24,6 +24,10 @@ int GetCPUInfoNoASM(CPUFeature feature)
   return 0;
 }
 
+uint64_t WebRtc_GetCPUFeaturesARM(void) {
+  return 0;
+}
+
 #if defined(WEBRTC_ARCH_X86_FAMILY)
 #ifndef _MSC_VER
 // Intrinsic for "cpuid".
